### DISCUSSION TITLE
disable, hidden, forced sofkeyboard using net maui

### DISCUSSION THREAD

**Discussion by persiabudi28** [dotnet/maui#4458] [Q&A]
how to hide to softkeyboarb when focus entry

**Comment by jsuarezruiz** [comment]
I think what you are missing is the implementation of the `IsFocused` property and the `Focus()` and `Unfocus()` methods. There are PRs to implement it and will be available in the upcoming version. 

![view-isfocused-ios](https://user-images.githubusercontent.com/6755973/153214678-f74ff55e-79eb-4f31-b165-7e9812e57adb.gif)

https://github.com/dotnet/maui/pull/4587

**Reply by persiabudi28** [reply]
like this https://theconfuzedsourcecode.wordpress.com/2017/05/19/a-keyboard-disabled-entry-control-in-xamarin-forms/

**Reply by persiabudi28** [reply]
hide softkeyboard very useful  like for barcode, virtual custom keyboard, etc

**Comment by Quaybe** [comment]
Ever figure this out?
